hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -467,6 +467,11 @@
   public static final TimeDuration OZONE_SCM_RATIS_MINIMUM_TIMEOUT_DEFAULT
       = TimeDuration.valueOf(1, TimeUnit.SECONDS);
 
+  public static final String OZONE_SCM_RATIS_REQUEST_TIMEOUT_KEY
+      = "ozone.scm.ratis.minimum.timeout";
```

Comment:
Should we add the key to `SCMHAConfiguration` ? 
BTW, the key `OZONE_SCM_RATIS_SERVER_REQUEST_TIMEOUT_KEY`, `OZONE_SCM_RATIS_SERVER_RETRY_CACHE_TIMEOUT_KEY`, `OZONE_SCM_RATIS_MINIMUM_TIMEOUT_KEY`, `OZONE_SCM_LEADER_ELECTION_MINIMUM_TIMEOUT_DURATION_KEY` etc are not used any more. Better clean up them.

Reply:
I think, we should be remove SCMHAConfiguration and pull all the required configs in ScmConfigKeys and remove the unsed ones. But, i would prefer to do this as a part of different jira altogether.

Reply:
Sure. Thanks for creating the jira.